lib/workers/repository/update/branch/handle-existing.ts

```diff
@@ -44,3 +51,50 @@ export async function handlepr(config: BranchConfig, pr: Pr): Promise<void> {
     logger.debug(`Merged PR with PrNo: ${pr.number} is blocking this branch`);
   }
 }
+
+export async function handleModifiedPr(
+  config: BranchConfig,
+  pr: Pr
+): Promise<void> {
+  if (config.suppressNotifications!.includes('prEditedNotification')) {
+    return;
+  }
+
+  const editedPrCommentTopic = 'Edited/Blocked Notification';
+  const content =
+    'Renovate will not automatically rebase this PR, because other commits have been found.\n' +
```

Comment:
```suggestion
    'Renovate will not automatically rebase this PR, because it does not recognize the last commit author and assumes somebody else may have edited the PR.\n' +
```